examples/plotting/spans_and_strips.py

```diff
@@ -1,3 +1,12 @@
+''' An example demonstration the use of `hspan`, `wspan`, and `vstrip`.
```

Comment:
typo on `wspan`, and ReST formatting actually requires *double* backticks. Also this phrasing would be more consistent with existing other examples:

```suggestion
''' This example demonstrates the use of ``hspan``, ``vspan``, ``hstrip``, and ``vstrip``.
```